examples/fineweb_edu_classifier_example.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import os
+
+os.environ["DASK_DATAFRAME__QUERY_PLANNING"] = "False"
+import time
+
+import torch
+from crossfit import op
+from crossfit.backend.torch.hf.model import HFModel
+from transformers import AutoConfig, AutoModelForSequenceClassification
+
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.utils.distributed_utils import get_client
+from nemo_curator.utils.script_utils import ArgumentHelper
+
+EDU_CLASSIFIER_PATH = "HuggingFaceTB/fineweb-edu-classifier"
+
+
+class FinewebEduClassifier(HFModel):
+    def __init__(self, path_or_name, max_mem_gb="16", autocast=False):
```

Comment:
The argparse has the type of `max_mem_gb` as an integer, while this has it as a string. If both are fine in crossfit, that's great, but I think we should be consistent in which one we use for Curator. I vote for integer, but if string is better for some reason go for that instead.

Reply:
Fixed